doc/source/examples/analysis/volumetric/linear_density.ipynb

```diff
@@ -68,7 +68,15 @@
      "shell.execute_reply": "2021-05-19T06:03:14.365692Z"
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #1.    plt.plot(np.linspace(0, 50, 200), density.results['x']['pos'])
</pre><p>Switch to the <code>results_density</code> alternative? </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/MDAnalysis/UserGuide/pull/241/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='14'/>